pages/getting-started/admin-console.md

```diff
@@ -5,14 +5,14 @@ description: Supercharge your day two operations.
 
 ## Introduction
 
-The Plural admin console is a web application that serves as a control panel for all your Plural applications. It:
+The Plural Admin Console is a web application that serves as a control panel for all your Plural applications. It:
 
 * manages automated upgrades delivered from the Kubernetes api
 * serves as a thin Grafana to visualize application metrics and logs
 * serves as a built-in K8s dashboard for all plural apps in the cluster, along with providing app-level health checking
 * is the touchpoint at which incidents can be filed with the owner of an application
 
-The console is not a strict dependency, but we highly recommended installing it. It can be installed and deployed like any other application on Plural.
+We highly recommended installing Plural Console. It can be installed and deployed like any other application on Plural.
```

Comment:
We might want to change the wording here to reflect the fact that Plural Console is currently required in our Cloud Shell onboarding.

Reply:
Yeah it's not required in CLI onboarding but can update

Reply:
Updated